code/modules/client/client_procs.dm

```diff
@@ -1,8 +1,8 @@
 	////////////
 	//SECURITY//
 	////////////
-#define UPLOAD_LIMIT		524288	//Restricts client uploads to the server to 0.5MB
-#define UPLOAD_LIMIT_ADMIN	2621440	//Restricts admin client uploads to the server to 2.5MB
+#define UPLOAD_LIMIT		524288		//Restricts client uploads to the server to 0.5MB
+#define UPLOAD_LIMIT_ADMIN	16777216	//Restricts admin client uploads to the server to 2.5MB
```

Comment:
Fix comment, it's no longer 2.5 MB it's 16 MB 
(Sorry for bikeshedding but the people need to know)